meta-delta/conf/distro/openbmc-fb.conf

```diff
@@ -0,0 +1,158 @@
+# Facebook OpenBMC distro
```

Comment:
Do we need this file as a second copy of the openbmc-delta.conf?

Reply:
No, it shall be removed here. I will update the pull request today. Thanks!